src/main/java/org/scijava/nativelib/NativeLoader.java

```diff
@@ -132,38 +128,17 @@
 	 *           <code>checkLink</code> method doesn't allow loading of the
 	 *           specified dynamic library
 	 */
-	public static void loadLibrary(final String libname, final String... searchPaths)
+	public static void loadLibrary(final String libName, final String... searchPaths)
```

Comment:
The spelling of `libName` also needs to be changed in the javadoc, to fix the javadoc errors in the Travis build.

Reply:
Thanks for pointing out the problem. I have fixed the typo in the new commit. This PR is ready for a code review, I assume.